src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/attachscanneddocs/AttachScannedDocsAboutToSubmitHandler.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.attachscanneddocs;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.NO;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Service
+public class AttachScannedDocsAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+        requireNonNull(callbackType, "callbacktype must not be null");
+
+        return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
+                && callback.getEvent() == EventType.ATTACH_SCANNED_DOCS
+                && callback.getCaseDetails() != null;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
+        final SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+
+        sscsCaseData.setEvidenceHandled(NO.getValue());
+        sscsCaseData.setHasUnprocessedAudioVideoEvidence(NO);
```

Comment:
Should this flag be worked out?
In a similar way as https://github.com/hmcts/sscs-tribunals-case-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/util/AudioVideoEvidenceUtil.java#L35

Reply:
fixed